libs/openFrameworks/utils/ofMatrixStack.cpp

```diff
@@ -24,6 +24,16 @@ ofMatrixStack::ofMatrixStack(const ofAppBaseWindow * window)
 ,currentMatrixMode(OF_MATRIX_MODELVIEW)
 ,currentMatrix(&modelViewMatrix)
```

Comment:
move currentMatrix() and flipRenderSurfaceMatrix() initializers at the end of initializers list to avoid reoder warnings